Makefile

```diff
@@ -36,4 +36,7 @@ test-unit:
 test-functional:
 	contrib/ci/test.sh ${IMAGE}
 
+test-functional-swift:
```

Comment:
@kmala Can we also add this new target to the `test` target so CI runs and vets this new test?